test/integration/transaction_test.rb

```diff
@@ -116,4 +116,35 @@ class TransactionTest < ActionDispatch::IntegrationTest
       assert page.has_no_selector?("#transaction_cross_domain_analytics", visible: :all)
     end
   end
+
+  context "locale is 'cy'" do
```

Comment:
Do we need tests for both 'sign in' and 'start now'?

Reply:
I was wondering about that, then I thought it might be overkill as an integration test?  Couldn't figure out a way to test it as a unit test.

Reply:
On government-frontend we do presenter level tests which would be more appropriate here I guess, not sure if this application has that concept though.

Reply:
I had a brainwave about presenter unit tests - now pushed 🚀